### PR TITLE
doc: Translate more strings

### PR DESCRIPTION
**Pull request by Playhi**
Translate more strings.
将更多内容进行了翻译（完善翻译）

## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)
    - Languages:
    - [ ] `en` English
    - [ ] `ko` Korean
    - [ ] `pt-br` Brazilian Portuguese
    - [ ] `ru` Russian
    - [ ] `th` Thai
    - [x] `zh-cn` simplified Chinese
    - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

File: source/zh-cn/docs/github-pages.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ The following instruction is adapted from [one-command deployment](/docs/one-com
     branch: gh-pages
   ```
 
-3. Run `hexo clean && hexo deploy`.
-4. Check the webpage at *username*.github.io.
+3. 执行 `hexo clean && hexo deploy`；
+4. 检查 *username*.github.io 上的网页。
```
Comment (SukkaW) [inline source/zh-cn/docs/github-pages.md]:
```suggestion
4. 检查 *username*.github.io  是否可以访问。
```

File: source/zh-cn/docs/github-pages.md [hunk 2]
```diff
@@ -45,10 +45,10 @@ deploy:
 
 ## Private repository
 
-The following instruction is adapted from [one-command deployment](/docs/one-command-deployment) page.
+以下教程改编自 [one-command deployment](/docs/one-command-deployment) 页面。
 
-1. Install [hexo-deployer-git](https://github.com/hexojs/hexo-deployer-git).
-2. Add the following configurations to **_config.yml**, (remove existing lines if any)
+1. 安装 [hexo-deployer-git](https://github.com/hexojs/hexo-deployer-git)；
+2. 将以下配置项添加到 **_config.yml**（删除现有行（如果有的话））；
```
[the remark this answers is not in the document]
Comment (Playhi) [reply source/zh-cn/docs/github-pages.md]: ###### _（看上去是国人/会中文，我就直接中文了哈）_
感觉两个翻译似乎都不是很好 :thinking: （有点纠结）

File: source/zh-cn/docs/github-pages.md [hunk 3]
```diff
@@ -45,10 +45,10 @@ deploy:
 
 ## Private repository
 
-The following instruction is adapted from [one-command deployment](/docs/one-command-deployment) page.
+以下教程改编自 [one-command deployment](/docs/one-command-deployment) 页面。
 
-1. Install [hexo-deployer-git](https://github.com/hexojs/hexo-deployer-git).
-2. Add the following configurations to **_config.yml**, (remove existing lines if any)
+1. 安装 [hexo-deployer-git](https://github.com/hexojs/hexo-deployer-git)；
```
[the remark this answers is not in the document]
Comment (Playhi) [reply source/zh-cn/docs/github-pages.md]: ###### _（看上去是国人/会中文，我就直接中文了哈）_
我在犹豫要不要分号 ；

[the remark this answers is not in the document]
Comment (SukkaW) [reply source/zh-cn/docs/github-pages.md]: 结尾可以用句号。对于列表（不论有序还是无序）结尾不宜用分号。

**Comment by stevenjoezhang** [comment]
感谢您的翻译！不过这里的翻译先合并进主线了： https://github.com/hexojs/site/pull/1360 不好意思😢

**Pull request closed without merging**